forte/processors/data_augment/algorithms/eda_processors.py

```diff
@@ -19,10 +19,11 @@
 implemented based on the ReplacementDataAugmentProcessor.
 """
 
+from math import ceil
 import random
-from typing import List, Tuple, Dict
+from typing import List, Dict
+
 from nltk.corpus import stopwords
```

Comment:
Let's don't depend on nltk directly, you can pass in the stop word dict as a dictionary. Here you also hardcoded "English", this is also not desired.

Reply:
Resolved by passing stopword to config.